proto/ibc/applications/middleware/fee/v1/query.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+package ibc.applications.middleware.fee.v1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "ibc/applications/middleware/fee/v1/fee.proto";
+import "google/api/annotations.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+// Query provides defines the gRPC querier service.
+service Query {
```

Comment:
relayers will want a service which queries all incentivized packets

Reply:
I added a service for this. Not sure if it's as you wanted. Also, not sure what we want in the request body -> https://github.com/cosmos/ibc-go/pull/272/files#diff-62029a85ec933b1fcec98cf25e2de0aa23db41b56a1cb8758b44dc6fcf806045R82

Reply:
I think just [optional pagination is good](https://github.com/cosmos/ibc-go/blob/0afa15af41997cfb4c5dd5f98de8043961e26195/proto/ibc/applications/transfer/v1/query.proto#L48)

Reply:
Added :+1: